models/ahrq_measures/pqi/staging/ahrq_measures__stg_pqi_inpatient_encounter.sql

```diff
@@ -15,7 +15,11 @@ select
   , patient_id
   , facility_id
   , paid_amount
-  , {{ date_part('YEAR', 'encounter_start_date') }} as year_number
+  {% if target.type == 'fabric' %}
+    , YEAR(encounter_start_date) as year_number
+  {% else %}
+    , {{ date_part('YEAR', 'encounter_start_date') }} as year_number
+  {% endif %}
```

Comment:
I can see date_part being used quite often. Instead of coding the syntax into every model, we could create our own version of this macro and include the fabric syntax. I believe we can keep the macro name and just add it to the Tuva package macros folder. I think our macro will take precedence over it.

Reply:
I will test this today!

Reply:
Should this be updated to remove the fabric logic since that was added to the date_part macro?